guides/common/modules/ref_supported-browsers.adoc

```diff
@@ -20,7 +20,7 @@ Other browsers may work unpredictably.
 endif::[]
 
 ifdef::satellite[]
-The {ProjectWebUI} and command-line interface support English, Simplified Chinese, Japanese, French.
+The {ProjectWebUI} and command-line interface support English, French, Japanese, Korean, and Simplified Chinese.
```

Comment:
I ended up putting Chinese last because of "Simplified". Any thoughts on whether it should be first or last?

Reply:
This works for me. I don't think this is the time and place to overthink things :) because the list is very short. However, you can also consider `Chinese (Simplified)` as an alternative, if you'd prefer to go by `C` when ordering the items.

Reply:
I kind of like English being the first in the list, but agreed we shouldn't overthink it because it's a very short list.